.kokoro

```diff
@@ -408,13 +408,12 @@ run_cocoapods() {
       bash scripts/test_all catalog/MDCCatalog.xcworkspace:MDCCatalog
     fi
   elif [ "$DEPENDENCY_SYSTEM" = "cocoapods-podspec" ]; then
-    # gem install cocoapods-deintegrate
-    # gem install cocoapods-clean
-    # cd catalog
-    # pod deintegrate
-    # pod install
-    # cd ../
-    bash scripts/prep_all
+    gem install cocoapods-deintegrate
```

Comment:
Please use the `gem_install` function in `scripts/lib/package_managers.sh` instead of `gem install`. They will guarantee gem gets updated before installing the tools.

Reply:
The functions are already imported into `.kokoro`, so you can use them here without any special effort.

Reply:
This is just my experiments to try and resolve the Podspec CI runner from failing for no good reason. Will probably not be taken care of in this PR.

Reply:
I see. Might be worth trying in a separate draft PR - this one is pretty noisy with all the reviewers attached. 😉